documents-webapp/src/main/webapp/vue-app/documents/components/body/views/FolderTreeViewDrawer.vue

```diff
@@ -25,7 +25,7 @@
             <v-list-item-title 
               class="body-2 mx-2 mt-1"
               :class="idItemActive === item.id ? 'primary--text font-weight-bold' : ''">
-              {{ item.name }}
+              {{ item.name === 'Documents' ? $t('documents.label.spaceHomeDocuments') : item.name }}               
```

Comment:
let's avoid expression inside the template if possible, could you add a computed property ?

Reply:
I used method instead, it can be done through computed(it is an item from array not from data, props..)